apps/src/templates/sectionAssessments/sectionAssessmentsRedux.js

```diff
@@ -214,12 +214,13 @@ export const getStudentMCResponsesForCurrentAssessment = (state) => {
     return {
       id: studentId,
       name: studentObject.student_name,
-      studentResponses: studentAssessment.level_results.map(answer => {
-        return {
-          responses: answer.student_result || '',
-          isCorrect: answer.status === 'correct',
-        };
-      })
+      studentResponses: studentAssessment.level_results.filter(answer => answer.status !== "free_response")
```

Comment:
could "free_response" be a constant somewhere?

Reply:
Great thought - going to follow up with another PR because there are about 8 places I need to update.